students/mielczarek_szymon/lesson_06_dicts_tuples_sets_args_kwargs/uppercase.py

```diff
@@ -0,0 +1,9 @@
+def capitalize(lower_case_word):
+    return chr(ord(lower_case_word[0]) - 0x20) + lower_case_word[1:]
```

Comment:
I would personally wrote is as:
`return lower_case_word[0].upper() + lower_case_word[1:]`

readability in the first place